appalerts/templates/appalerts/alert_list.html

```diff
@@ -23,21 +23,50 @@
     .sodar-app-alert-row div:nth-child(1) {
         max-width: 40px;
     }
+
   </style>
 {% endblock css %}
 
 {% block projectroles %}
 
 <div class="row sodar-subtitle-container">
-  <h2><i class="iconify" data-icon="mdi:alert-octagram"></i> App Alerts</h2>
-  <a class="btn btn-primary ml-auto sodar-app-alert-btn-dismiss
-           {% if object_list.count == 0 %}disabled{% endif %}"
-     id="sodar-app-alert-btn-dismiss-all"
-     role="button"
-     data-uuid=""
-     data-dismiss-url="{% url 'appalerts:ajax_dismiss_all' %}">
-    <i class="iconify" data-icon="mdi:close-thick"></i> Dismiss All
-  </a>
+  {% if dismissed %}
+  <h2><i class="iconify" data-icon="mdi:alert-octagram-outline"></i>
```

Comment:
Some small cleanup needed here and within the `{% if not dismissed %}` block for indentation.